docs/semgrep-ci/configuration-reference.md

```diff
@@ -22,20 +22,27 @@ SEMGREP_RULES="p/security-audit p/secrets"
 
 ## Diff-aware scanning (`SEMGREP_BASELINE_REF`)
 
-For [diff-aware scans](overview.md#features), set this variable
-to the git ref (branch name, tag, or commit hash) to use as a baseline.
-For example, to report findings newly added
-since branching off from your `main` branch, set
+For [diff-aware scans](overview.md#features), set this variable to create a baseline. This option limits scan results to those introduced after the git commit, in a branch, or tag. For example, you have a repository with 10 commits. You set the commit number 8 as the baseline. Consequently, Semgrep only returns scan results introduced by changes in commits 9 and 10.
 
-```sh
-SEMGREP_BASELINE_REF=main
-```
+NOTE: It is best to perform diff scans on branches other than your `main` branch. Performing a diff scan after a full-project scan of the same branch closes findings of the full-project. All findings of a particular branch are marked as fixed when the branch is re-scanned. As a consequence, Semgrep no longer reports their findings.
+
+### Examples of `SEMGREP_BASELINE_REF`
+
+To only report findings newly added
+since branching off from your `main` branch, set the following:
+    ```sh
+    SEMGREP_BASELINE_REF=$(git merge-base main HEAD)
+    ```
 
-NOTE: performing a diff scan after a full-project scan of the same branch will close the full-project scan's findings. This is because all findings are marked as fixed if the branch they are on is re-scanned and they are no longer reported. Therefore, it is best to perform diff scans on branches other than your `main` branch.
+To only report findings newly added
+after a specific commit, set the following:
+    ```sh
+    SEMGREP_BASELINE_REF=<GIT_COMMIT_HASH>
```

Comment:
@underyx I should have asked before. Are the `<>` part of CI syntax or do you mean it just as a placeholder? Because we can use Google style guide ideas about placeholders and maybe color code it. :-) https://developers.google.com/style/placeholders?hl=en

Reply:
it's not a part of the CI syntax! Now having read the style guide; your version was better, and it should be color coded.